.github/workflows/code-quality.yml

```diff
@@ -6,18 +6,30 @@ on:
     branches:
       - main
 
+env:
+  NODE_VERSION: 21.x
```

Comment:
Let's use the LTS version
https://endoflife.date/nodejs
![image](https://github.com/rolling-scopes/site/assets/69504277/bb4928f6-c712-451d-b0a1-dbbb90834aa4)

Reply:
Done